src/main/java/core/org/akaza/openclinica/service/StudyParticipantServiceImpl.java

```diff
@@ -45,6 +45,7 @@
 import java.io.File;
 import java.io.IOException;
 import java.io.PrintWriter;
+import java.text.MessageFormat;
```

Comment:
Delete this import as you are not using it in this class

Reply:
done